class/defaults.yml

```diff
@@ -11,6 +11,30 @@ parameters:
         repository: kyverno/kyvernopre
         version: ${kyverno:images:kyverno:version}
 
+    manifest_version: ${kyverno:images:kyverno:version}
+    manifest_url: https://raw.githubusercontent.com/kyverno/kyverno/${kyverno:manifest_version}/definitions
+
+    replicas: 3
+
+    podDisruptionBudget:
+      minAvailable: 1
+
+    # One of 'master', 'infra', null
+    nodeSelectorRole: master
```

Comment:
@simu @corvus-ch Is it alright to let kyverno run on `master` nodes? IMO it becomes part of the k8s API with all those validation rules.

Reply:
Should be fine I think. Not sure what the official stance of OpenShift 4.x is regarding running additional pods on the masters.